clients/client-core/src/config/mod.rs

```diff
@@ -247,8 +247,8 @@ impl<T: NymConfig> Config<T> {
         self.debug.disable_main_poisson_packet_distribution
     }
 
-    pub fn get_use_extended_packet_size(&self) -> bool {
-        self.debug.use_extended_packet_size
+    pub fn get_use_extended_packet_size(&self) -> Option<String> {
```

Comment:
I'd suggest it should then return an enum : )
You should take a look at one of the enums in our wallet types (unfortunately I don't remember the name); it has some derived trait that allows for serde from/to string